.github/in-cluster-test-scripts/aks.sh

```diff
@@ -14,5 +14,8 @@ sleep 10s
 # Run connectivity test
 cilium connectivity test --debug --all-flows
 
+# Run performance test
+cilium connectivity test --perf --duration 1
```

Comment:
As I understand the current logic, this would also run the connectivity tests so we would effectively run them twice. I'd either fold in the `--perf` into the command above or change `--perf` behavior to only run the performance tests. Likewise for the other in-cluster test scripts.

Reply:
actually it will just run the perf test. Instead of double dipping.

Reply:
Ah, right. Sorry for the noise.